functions/AuthUtils.js

```diff
@@ -0,0 +1,186 @@
+const { Issuer } = require('openid-client');
+const { generators } = require('openid-client');
+const jwt = require('jsonwebtoken');
+const cookie = require('cookie');
+
+const getOpenIDClient = async () => {
+    const issuer = await Issuer.discover(`https://${process.env.AUTH0_DOMAIN}`);
+    return new issuer.Client({
+        client_id: process.env.AUTH0_CLIENT_ID,
+        redirect_uris: [`${process.env.URL}/.netlify/functions/callback`],
+        response_types: ['id_token'],
+    });
+};
+
+const generateNetlifyJWT = async (tokenData) => {
+    const iat = Math.floor(Date.now() / 1000);
+    const twoWeeksInSeconds = 14 * 24 * 3600;
```

Comment:
Nit: Could be a top-level constant

Reply:
Good suggestion. I made it more generic NETLIFY_TOKEN_EXPIRATION_SECONDS also